### PR TITLE
SG-34298 - Rebranding

### PR DESCRIPTION
**Pull request by juanburgosautoglb**
Change all the shotgrid/shotgun occurrences in the repository.

### CODE REVIEW

**Review by staceyoue — COMMENTED** [review]
@juanburgosautoglb why was the icon changed from the Alias icon to the Canadian flag?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 Toolkit framework for integration with Autodesk Alias.
 
-The main use of this framework is by the [Alias Engine](https://github.com/shotgunsoftware/tk-alias). The framework allows the engine to run alongside Alias, providing access to the ShotGrid Toolkit Apps that can interact wtih the data in Alias.
+The main use of this framework is by the [Alias Engine](https://github.com/shotgunsoftware/tk-alias). The framework allows the engine to run alongside Alias, providing access to the Flow Production Tracking Apps that can interact wtih the data in Alias.
```
Comment (julien-lang) [inline README.md:9]:
```suggestion
The main use of this framework is by the [Alias Engine](https://github.com/shotgunsoftware/tk-alias). The framework allows the engine to run alongside Alias, providing access to the Flow Production Tracking Toolkit Apps that can interact with the data in Alias.
```

File: dev/README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ The `get-pip.py` script is not included to set up the embeddable Python packages
 
 ## Required Python Packages
 
-The framework requires additional Python packages to the standard package that ShotGrid Toolkit ships with. To make these packages available to the framework, they must be included in the distribution directory `dist/Python/Python<MAJOR_VERSION><MINOR_VERSION>/packages`.
+The framework requires additional Python packages to the standard package that Flow Production Tracking ships with. To make these packages available to the framework, they must be included in the distribution directory `dist/Python/Python<MAJOR_VERSION><MINOR_VERSION>/packages`.
```
Comment (julien-lang) [inline dev/README.md:72]:
```suggestion
The framework requires additional Python packages to the standard package that Flow Production Tracking Toolkit ships with. To make these packages available to the framework, they must be included in the distribution directory `dist/Python/Python<MAJOR_VERSION><MINOR_VERSION>/packages`.
```

File: dist/README.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ If the embeddable package needs to be updated, for example, to add more site-pac
 
 ## Required Python Packages
 
-Additional Python packages are required to run the framework that ShotGrid Toolkit does not ship with. These packages are added to the Python system path at run time [here](https://github.com/shotgunsoftware/tk-framework-alias/blob/develop/python/tk_framework_alias/__init__.py#L11-L20). By including the packages in the path, these modules must be made available by either:
+Additional Python packages are required to run the framework that Flow Production Tracking does not ship with. These packages are added to the Python system path at run time [here](https://github.com/shotgunsoftware/tk-framework-alias/blob/develop/python/tk_framework_alias/__init__.py#L11-L20). By including the packages in the path, these modules must be made available by either:
```
Comment (julien-lang) [inline dist/README.md:75]:
```suggestion
Additional Python packages are required to run the framework that Flow Production Tracking Toolkit does not ship with. These packages are added to the Python system path at run time [here](https://github.com/shotgunsoftware/tk-framework-alias/blob/develop/python/tk_framework_alias/__init__.py#L11-L20). By including the packages in the path, these modules must be made available by either:
```

**Review by julien-lang — CHANGES_REQUESTED** [review]

**Review by julien-lang — APPROVED** [review]

**Pull request merged**